Apps/fastlane/Fastfile

```diff
@@ -79,8 +64,8 @@ platform :ios do
       build_path: "build", # Save derived data to Apps/X/build folder. CI will parse this folder later for tracking SDK size. 
       cloned_source_packages_path: "spm_packages", # Save SPM dependencies to Apps/X/spm_packages folder. We can cache this folder to speed up builds on CI. 
     )
-
-    are_environment_variables_set_for_build_uploading = !ENV["FIREBASE_APP_DISTRIBUTION_SERVICE_ACCOUNT_CREDS_B64"].empty?
+    
```

Comment:
I noticed that `.empty?` was throwing ruby errors when running. Not sure why this is happening now when this has been working for a while now. This change is a bug fix.